src/pages/Article/Article.js

```diff
@@ -2,7 +2,111 @@ import React from 'react';
 import styled from 'styled-components';
 
 const Article = () => {
-  return <p>hello</p>;
+  return (
+    <div>
```

Comment:
단순히 최상단을 묶어주기 위한 div라면 Fragment를 활용하시면 되겠네요!
```suggestion
    <>
```